solidity/test/bridge/Bridge.test.ts

```diff
@@ -269,6 +269,108 @@ describe("Bridge", () => {
     })
   })
 
+  describe.only("updateRedemptionParameters", () => {
```

Comment:
`.only` 😬

Reply:
Removed in https://github.com/keep-network/tbtc-v2/pull/224/commits/30cfb21bcebe111cafd3696953809f19e4528e05.